Cakefile

```diff
@@ -80,6 +80,11 @@ monthNames = [
   'December'
 ]
 
+harmonyFiles = [
```

Comment:
This variable is never referenced. I would think we can remove this whole block.